client/app/admin/create-project/create-project.html

```diff
@@ -228,13 +238,22 @@ <h1 class="section__title">
                                   ng-show="createProjectCtrl.projectNameForm.submitted && createProjectCtrl.projectNameForm.projectName.$error.required">
                                 Required
                             </span>
-                            <div ng-show="createProjectCtrl.createProjectFail"
+                            <div ng-show="createProjectCtrl.createProjectFailReason == 400"
+                                 class="alert alert--danger" role="alert">
+                                <p><strong>Error:</strong> AOI geometry must be a valid multipolygon
+                                </p>
+                            </div>
+                            <div ng-show="createProjectCtrl.createProjectFailReason == 500"
+                                 class="alert alert--danger" role="alert">
+                                <p><strong>Error:</strong> Project could not be created.
+                                </p>
+                            </div>
+                            <div ng-show="createProjectCtrl.createProjectFail == -1"
```

Comment:
Should this be createProjectCtrl.createProjectFailReason instead of createProjectCtrl.createProjectFail?